Wikipedia/Code/WikidataDescriptionEditingController.swift

```diff
@@ -39,14 +39,35 @@ enum WikidataPublishingError: LocalizedError {
 }
 
 @objc public final class WikidataDescriptionEditingController: NSObject {
-    private var blacklistedLanguages = Set<String>()
+    weak var viewContext: NSManagedObjectContext?
+
+    @objc public init(with viewContext: NSManagedObjectContext) {
+        self.viewContext = viewContext
+    }
+
+    private let BlacklistedLanguagesKey = "WMFWikidataDescriptionEditingBlacklistedLanguagesKey"
+    private var blacklistedLanguages: NSSet {
+        assert(Thread.isMainThread)
+        let fallback = NSSet(set: ["en"])
+        guard
+            let viewContext = viewContext,
+            let keyValue = viewContext.wmf_keyValue(forKey: BlacklistedLanguagesKey),
+            let value = keyValue.value as? NSSet else {
+                assertionFailure("Failed to retrieve WMFKeyValue for key \(BlacklistedLanguagesKey), returning fallback object \(fallback)")
```

Comment:
Is this assertion failure necessary if there's a fallback? Since the value isn't set by a library migration, it won't be there until the initial remote config check completes

Reply:
yeah that's a good point 👍